<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertOrEditUsersPage</class>
 <widget class="QMainWindow" name="InsertOrEditUsersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="previousPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(200, 0, 0);</string>
    </property>
    <property name="text">
     <string>Հետ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Պահպանել</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>360</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Նոր օգտվող</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabe">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Օգտանուն</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameLineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Գաղտնաբառ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Տեսակ</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Գործող</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameLineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Անուն</string>
    </property>
   </widget>
   <widget class="QLabel" name="surnameLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Ազգանուն</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeComboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Ադմինիստրատոր</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Գանձապահ</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="stateComboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Ակտիվ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Պասիվ</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="addressLineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(39, 51, 59);</string>
    </property>
    <property name="text">
     <string>Հասցե</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
